gulpfile.babel.js

```diff
@@ -538,169 +229,46 @@ gulp.task('zip:mdl', () => {
     ])
     .pipe($.zip('mdl.zip'))
     .pipe(gulp.dest('dist'));
-});
-
-/**
- * Returns the list of children directories inside the given directory.
- * @param {string} dir the parent directory
- * @return {Array<string>} list of child directories
- */
-function getSubDirectories(dir) {
-  return fs.readdirSync(dir)
-    .filter(file => fs.statSync(path.join(dir, file)).isDirectory());
 }
 
-// Generate release archives containing the templates and assets for templates.
-gulp.task('zip:templates', () => {
-  const templates = getSubDirectories('dist/templates');
-
-  // Generate a zip file for each template.
-  const generateZips = templates.map(template => {
-    return gulp.src([
-        `dist/templates/${template}/**/*.*`,
-        'LICENSE'
-      ])
-      .pipe($.rename(path => {
-        path.dirname = path.dirname.replace(`dist/templates/${template}`, '');
-      }))
-      .pipe($.zip(`${templateArchivePrefix}${template}.zip`))
-      .pipe(gulp.dest('dist'));
-  });
-
-  return mergeStream(generateZips);
-});
-
-gulp.task('zip', [
-  'zip:templates',
-  'zip:mdl'
-]);
-
-gulp.task('genCodeFiles', () => {
-  return gulp.src([
-      'dist/material.*@(js|css)?(.map)',
-      'dist/mdl.zip',
-      `dist/${templateArchivePrefix}*.zip`
-    ], {read: false})
-    .pipe($.tap(file => {
-      codeFiles += ` dist/${path.basename(file.path)}`;
-    }));
-});
+function emptyStream() {
```

Comment:
I generally ran into the problem that utility functions are not distinguishable from actual tasks (more visible in the getmdl.io repo, which has the hacky implementation of a jekyll-like system). Any input on this is appreciated.